plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/BaseTrinoCatalogTest.java

```diff
@@ -169,6 +174,71 @@ public void testCreateTable()
         }
     }
 
+    @Test
```

Comment:
```suggestion
@Test
    public void testCreateWithSortTable()
            throws Exception
    {
        TrinoCatalog catalog = createTrinoCatalog(false);
        String namespace = "test_create_sort_table_" + randomNameSuffix();
        String table = "tableName";
        SchemaTableName schemaTableName = new SchemaTableName(namespace, table);
        catalog.createNamespace(SESSION, namespace, defaultNamespaceProperties(namespace), new TrinoPrincipal(PrincipalType.USER, SESSION.getUser()));
        try {
            Schema tableSchema = new Schema(Types.NestedField.of(1, true, "col1", Types.LongType.get()),
                    Types.NestedField.of(2, true, "col2", Types.StringType.get()),
                    Types.NestedField.of(3, true, "col3", Types.TimestampType.withZone()),
                    Types.NestedField.of(4, true, "col4", Types.StringType.get()));

            SortOrder sortOrder = SortOrder.builderFor(tableSchema)
                    .asc("col1")
                    .desc("col2", NullOrder.NULLS_FIRST)
                    .desc("col3")
                    .desc(Expressions.year("col3"), NullOrder.NULLS_LAST)
                    .desc(Expressions.month("col3"), NullOrder.NULLS_FIRST)
                    .asc(Expressions.day("col3"), NullOrder.NULLS_FIRST)
                    .asc(Expressions.hour("col3"), NullOrder.NULLS_FIRST)
                    .desc(Expressions.bucket("col2", 10), NullOrder.NULLS_FIRST)
                    .desc(Expressions.truncate("col4", 5), NullOrder.NULLS_FIRST).build();
            catalog.newCreateTableTransaction(
                            SESSION,
                            schemaTableName,
                            tableSchema,
                            PartitionSpec.unpartitioned(),
                            sortOrder,
                            arbitraryTableLocation(catalog, SESSION, schemaTableName),
                            ImmutableMap.of())
                    .commitTransaction();
            assertThat(catalog.listTables(SESSION, Optional.of(namespace))).contains(schemaTableName);
            assertThat(catalog.listTables(SESSION, Optional.empty())).contains(schemaTableName);

            Table icebergTable = catalog.loadTable(SESSION, schemaTableName);
            assertEquals(icebergTable.name(), quotedTableName(schemaTableName));
            assertEquals(icebergTable.schema().columns().size(), 4);
            assertEquals(icebergTable.schema().columns().get(0).name(), "col1");
            assertEquals(icebergTable.schema().columns().get(0).type(), Types.LongType.get());
            assertEquals(icebergTable.schema().columns().get(1).name(), "col2");
            assertEquals(icebergTable.schema().columns().get(1).type(), Types.StringType.get());
            assertEquals(icebergTable.schema().columns().get(2).name(), "col3");
            assertEquals(icebergTable.schema().columns().get(2).type(), Types.TimestampType.withZone());
            assertEquals(icebergTable.schema().columns().get(3).name(), "col4");
            assertEquals(icebergTable.schema().columns().get(3).type(), Types.StringType.get());
            assertEquals(icebergTable.sortOrder(), sortOrder);

            catalog.dropTable(SESSION, schemaTableName);
        }
        finally {
            if (!catalog.listTables(SESSION, Optional.of(schemaTableName.getSchemaName())).isEmpty()) {
                catalog.dropTable(SESSION, schemaTableName);
            }
            catalog.dropNamespace(SESSION, namespace);
        }
    }
```


Test is failing otherwise when working with Hive metastore

```
2023-01-06T17:25:56.4534678Z [ERROR] io.trino.plugin.iceberg.catalog.hms.TestTrinoHiveCatalogWithHiveMetastore.testCreateWithSortTable  Time elapsed: 3.776 s  <<< FAILURE!
2023-01-06T17:25:56.4535707Z java.lang.IllegalArgumentException: java.net.UnknownHostException: hadoop-master
2023-01-06T17:25:56.4537083Z 	at org.apache.hadoop.security.SecurityUtil.buildTokenService(SecurityUtil.java:447)
2023-01-06T17:25:56.4537804Z 	at org.apache.hadoop.hdfs.NameNodeProxiesClient.createProxyWithClientProtocol(NameNodeProxiesClient.java:131)
2023-01-06T17:25:56.4538407Z 	at org.apache.hadoop.hdfs.DFSClient.<init>(DFSClient.java:355)
2023-01-06T17:25:56.4538816Z 	at org.apache.hadoop.hdfs.DFSClient.<init>(DFSClient.java:289)
2023-01-06T17:25:56.4539331Z 	at org.apache.hadoop.hdfs.DistributedFileSystem.initialize(DistributedFileSystem.java:176)
2023-01-06T17:25:56.4539910Z 	at io.trino.hdfs.TrinoFileSystemCache.createFileSystem(TrinoFileSystemCache.java:167)
2023-01-06T17:25:56.4540454Z 	at io.trino.hdfs.TrinoFileSystemCache.getInternal(TrinoFileSystemCache.java:123)
2023-01-06T17:25:56.4540960Z 	at io.trino.hdfs.TrinoFileSystemCache.get(TrinoFileSystemCache.java:91)
2023-01-06T17:25:56.4541473Z 	at org.apache.hadoop.fs.ForwardingFileSystemCache.get(ForwardingFileSystemCache.java:38)
2023-01-06T17:25:56.4541961Z 	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:479)
2023-01-06T17:25:56.4542369Z 	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:361)
2023-01-06T17:25:56.4542815Z 	at io.trino.hdfs.HdfsEnvironment.lambda$getFileSystem$0(HdfsEnvironment.java:74)
2023-01-06T17:25:56.4543348Z 	at io.trino.hdfs.authentication.NoHdfsAuthentication.doAs(NoHdfsAuthentication.java:25)
2023-01-06T17:25:56.4543871Z 	at io.trino.hdfs.HdfsEnvironment.getFileSystem(HdfsEnvironment.java:73)
2023-01-06T17:25:56.4544330Z 	at io.trino.hdfs.HdfsEnvironment.getFileSystem(HdfsEnvironment.java:67)
2023-01-06T17:25:56.4544851Z 	at io.trino.filesystem.hdfs.HdfsFileSystem.deleteDirectory(HdfsFileSystem.java:116)
2023-01-06T17:25:56.4545735Z 	at io.trino.plugin.iceberg.catalog.AbstractTrinoCatalog.deleteTableDirectory(AbstractTrinoCatalog.java:185)
2023-01-06T17:25:56.4546415Z 	at io.trino.plugin.iceberg.catalog.hms.TrinoHiveCatalog.dropTable(TrinoHiveCatalog.java:321)
2023-01-06T17:25:56.4547108Z 	at io.trino.plugin.iceberg.catalog.BaseTrinoCatalogTest.testCreateWithSortTable(BaseTrinoCatalogTest.java:232)
2023-01-06T17:25:56.4547757Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2023-01-06T17:25:56.4548338Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2023-01-06T17:25:56.4548977Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2023-01-06T17:25:56.4549509Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
2023-01-06T17:25:56.4550013Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2023-01-06T17:25:56.4550522Z 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
2023-01-06T17:25:56.4550959Z 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
2023-01-06T17:25:56.4551417Z 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
2023-01-06T17:25:56.4551917Z 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
2023-01-06T17:25:56.4552431Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
2023-01-06T17:25:56.4552951Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2023-01-06T17:25:56.4553577Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2023-01-06T17:25:56.4554006Z 	at java.base/java.lang.Thread.run(Thread.java:833)
2023-01-06T17:25:56.4554566Z Caused by: java.net.UnknownHostException: hadoop-master
2023-01-06T17:25:56.4554848Z 	... 32 more
```